packages/cyberstorm/src/components/AntiResult/AntiResultMessage.tsx

```diff
@@ -0,0 +1,19 @@
+import { ReactNode } from "react";
+import styles from "./AntiResult.module.css";
+import { classnames } from "../../utils/utils";
+
+interface Props {
+  children?: ReactNode | ReactNode[];
+  className?: string;
+}
+
+export function AntiResultMessage(props: Props) {
```

Comment:
How likely do you or grav find it that we're going to use different styles for the title or the message? I'd think consistently forcing the styles defined the "AntiResult" component could be what we actually want? In that case the title and the message could be passed to the main component as plain text props? That would make the usage less cumbersome.